solutions/standard/stack_definition.json

```diff
@@ -701,6 +701,10 @@
           "name": "add_container_name_suffix",
           "value": true
         },
+        {
+          "name": "cc_pipeline_properties",
+          "value": "[{\"pipeline_id\":\"cc\",\"properties\":[{\"name\":\"app-name\",\"type\":\"text\",\"value\":\"rag-sample-app\"},{\"name\":\"cos-api-key\",\"type\":\"secure\",\"value\":\"\"},{\"name\":\"cos-bucket-name\",\"type\":\"text\",\"value\":\"\"},{\"name\":\"cos-endpoint\",\"type\":\"text\",\"value\":\"\"},{\"name\":\"cra-bom-generate\",\"type\":\"single_select\",\"value\":\"1\",\"enum\":\"[0,1]\"},{\"name\":\"cra-deploy-analysis\",\"type\":\"single_select\",\"value\":\"1\",\"enum\":\"[0,1]\"},{\"name\":\"cra-vulnerability-scan\",\"type\":\"single_select\",\"value\":\"1\",\"enum\":\"[0,1]\"},{\"name\":\"doi-environment\",\"type\":\"text\",\"value\":\"\"},{\"name\":\"doi-ibmcloud-api-key\",\"type\":\"secure\",\"value\":\"ibmcloud-api-key\"},{\"name\":\"doi-toolchain-id\",\"type\":\"text\",\"value\":\"\"},{\"name\":\"environment-tag\",\"type\":\"text\",\"value\":\"\"},{\"name\":\"event-notifications\",\"type\":\"text\",\"value\":\"0\"},{\"name\":\"git-token\",\"type\":\"secure\",\"value\":\"\"},{\"name\":\"ibmcloud-api-key\",\"type\":\"secure\",\"value\":\"\"},{\"name\":\"opt-in-auto-close\",\"type\":\"text\",\"value\":\"1\"},{\"name\":\"opt-in-cra-auto-remediation\",\"type\":\"text\",\"value\":\"true\"},{\"name\":\"opt-in-cra-auto-remediation-enabled-repos\",\"type\":\"text\",\"value\":\"\"},{\"name\":\"opt-in-cra-auto-remediation-force\",\"type\":\"text\",\"value\":\"true\"},{\"name\":\"opt-in-dynamic-api-scan\",\"type\":\"text\",\"value\":\"\"},{\"name\":\"opt-in-dynamic-scan\",\"type\":\"text\",\"value\":\"1\"},{\"name\":\"opt-in-dynamic-ui-scan\",\"type\":\"text\",\"value\":\"1\"},{\"name\":\"opt-in-gosec\",\"type\":\"text\",\"value\":\"\"},{\"name\":\"opt-in-sonar\",\"type\":\"text\",\"value\":\"true\"},{\"name\":\"peer-review-compliance\",\"type\":\"text\",\"value\":\"1\"},{\"name\":\"pipeline-config\",\"type\":\"text\",\"value\":\".pipeline-config.yaml\"},{\"name\":\"pipeline-config-branch\",\"type\":\"text\",\"value\":\"\"},{\"name\":\"pipeline-debug\",\"type\":\"single_select\",\"value\":\"0\",\"enum\":\"[0,1]\"},{\"name\":\"slack-notifications\",\"type\":\"text\",\"value\":\"0\"},{\"name\":\"sonarqube-config\",\"type\":\"text\",\"value\":\"default\"}]}]"
```

Comment:
I think we may need to set pipeline-config-branch to "main" - we are seeing some issues with CD pipeline on ROKS deployment.

Reply:
Updated with that value to see if it fixes your clone issue - also added entire CD pipeline parameters for ROKS, that was missing.